WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutHeaderView.swift

```diff
@@ -1,16 +1,37 @@
 import Foundation
-import SwiftUI
+
+
+/// Defines the content of the header that appears on the top level about screen.
+struct AboutScreenAppInfo {
+    /// The app's name
+    let name: String
+    /// The current build version of the app
+    let version: String
+    /// The app's icon
+    let icon: UIImage
+}
+
+struct AboutScreenFonts {
+    let appName: UIFont
+    let appVersion: UIFont
+
+    static let defaultFonts: AboutScreenFonts = {
+        // Title is serif semibold large title
+        let fontDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: .largeTitle)
```

Comment:
I don't love all this font wrangling here, but it's a tricky font to construct and I don't think we want to necessarily go down the route of adding UIFont extensions to this package?

Reply:
👍